chapter_post_and_database.asciidoc

```diff
@@ -1690,6 +1718,7 @@ Ran 1 test in 5.036s
 
 OK
 ----
+// CSANAD:  The test will actually fail because of `self.fail("Finish the test!")`
```

Comment:
hmmm i'll need to check this.

Reply:
I'll double check later tonight but I can't recall commenting that line out in this version of the book yet.

Reply:
Ok, so what I found is this `self.fail` is still there in chapter_philosophy_and_refactoring
https://github.com/hjwp/book-example/blob/3d1c41518597cc43f18c1d8a8ac86967998ebc3b/functional_tests.py

But it's been removed from chapter_post_and_database
https://github.com/hjwp/book-example/blob/335a69f69ec9dfac6fe567dcab47cbfbcbc27968/functional_tests.py

I couldn't find this change in either of the chapters' text though.

Reply:
ok the change is in the listing here at ch05l015.  i'll change the text to make it clearer that we are deleting the self.fail.  thanks buddy.